modules/arm_plugin/armplg_build.sh

```diff
@@ -149,33 +149,12 @@ cmake -DInferenceEngineDeveloperPackage_DIR=$OPENVINO_HOME/build \
       -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DENABLE_DATA=OFF \
```

Comment:
Should `${PYTHONVER}m` be replaced here as well?

Reply:
Fixed